org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/AbstractLanguageServerTest.xtend

```diff
@@ -42,6 +42,7 @@ import org.eclipse.lsp4j.DocumentSymbol
 import org.eclipse.lsp4j.DocumentSymbolParams
 import org.eclipse.lsp4j.FileChangeType
 import org.eclipse.lsp4j.FileEvent
+import org.eclipse.lsp4j.FoldingRangeRequestParams
```

Comment:
Please also update the year in the [copyright](https://github.com/eclipse/xtext-core/blob/bc72a8f56082ee1bc6ff9fc0f14f379b6a3c4318/org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/AbstractLanguageServerTest.xtend#L2) of the AbstractLanguageServerTest class.

Reply:
Thank for fixing the copyright header in the `AbstractLanguageServerTest.xtend` file. Please also fix it in the generated `AbstractLanguageServerTest.java` file.

Reply:
Oops, looks like I didn't pay attention there. Thanks for the heads-up.